modules/govuk/manifests/app/config.pp

```diff
@@ -139,6 +145,9 @@
       "${title}-SENTRY_DSN":
         varname => 'SENTRY_DSN',
         value   => $sentry_dsn;
+      "${title}-GDS_API_DISABLE_CACHE":
+        varname => 'GDS_API_DISABLE_CACHE',
+        value   => "${!cache_api_calls}";
```

Comment:
If I understand [how it decides whether to disable the cache](https://github.com/alphagov/gds-api-adapters/blob/6d1576dd0340720fb7bdedabec08b8fea64125b3/lib/gds_api/json_client.rb#L42) correctly, a string `"false"` will be truthy, should it be that the environment variable is either `""` or `"true"` rather than `"false"` or `"true"`?

Reply:
`""` is truthy too..

Reply:
Perhaps that env var override never worked.

Reply:
Ah that's where Ruby and Python differ about the truthness of strings 😉

Reply:
So what shall we do?  If we want to take this approach we'd have to release a new version of the gem and bump it in every app.

Reply:
Can we do something like:

```puppet
if $cache_api_calls {
  # ensure present the environment variable with the value
} else {
  # ensure absent the environment variable
}
```

Reply:
(as a bit of an FYI then, in Ruby only `nil` and `false` are falsey)

Reply:
We can, yes, though that wouldn't override any where it's explicitly turned on in the app code, though I don't *think* there are any cases of that.